packages/plugin-jsdocs/src/lib/runner/models.ts

```diff
@@ -0,0 +1,55 @@
+import { SyntaxKind } from 'ts-morph';
+
+/** The possible coverage types for documentation analysis */
+export const SINTAX_KIND_LITERAL_STRING = {
+  CLASSES: 'classes',
+  METHODS: 'methods',
+  FUNCTIONS: 'functions',
+  INTERFACES: 'interfaces',
+  ENUMS: 'enums',
+  VARIABLES: 'variables',
+  PROPERTIES: 'properties',
+  TYPES: 'types',
+} as const;
+
+export type CoverageType =
+  (typeof SINTAX_KIND_LITERAL_STRING)[keyof typeof SINTAX_KIND_LITERAL_STRING];
+
+/** Maps the SyntaxKind from the library ts-morph to the coverage type. */
+export const SYNTAX_COVERAGE_MAP = new Map<SyntaxKind, CoverageType>([
+  [SyntaxKind.ClassDeclaration, SINTAX_KIND_LITERAL_STRING.CLASSES],
+  [SyntaxKind.MethodDeclaration, SINTAX_KIND_LITERAL_STRING.METHODS],
+  [SyntaxKind.FunctionDeclaration, SINTAX_KIND_LITERAL_STRING.FUNCTIONS],
+  [SyntaxKind.InterfaceDeclaration, SINTAX_KIND_LITERAL_STRING.INTERFACES],
+  [SyntaxKind.EnumDeclaration, SINTAX_KIND_LITERAL_STRING.ENUMS],
+  [SyntaxKind.VariableDeclaration, SINTAX_KIND_LITERAL_STRING.VARIABLES],
+  [SyntaxKind.VariableStatement, SINTAX_KIND_LITERAL_STRING.VARIABLES],
+  [SyntaxKind.PropertyDeclaration, SINTAX_KIND_LITERAL_STRING.PROPERTIES],
+  [SyntaxKind.TypeAliasDeclaration, SINTAX_KIND_LITERAL_STRING.TYPES],
+]);
```

Comment:
Since you don't actually need `SINTAX_KIND_LITERAL_STRING` in runtime, you can simplify this with a string union:

```suggestion
/** The possible coverage types for documentation analysis */
export type CoverageType =
  | 'classes'
  | 'methods'
  | 'functions'
  | 'interfaces'
  | 'enums'
  | 'variables'
  | 'properties'
  | 'types';

/** Maps the SyntaxKind from the library ts-morph to the coverage type. */
export const SYNTAX_COVERAGE_MAP = new Map<SyntaxKind, CoverageType>([
  [SyntaxKind.ClassDeclaration, 'classes'],
  [SyntaxKind.MethodDeclaration, 'methods'],
  [SyntaxKind.FunctionDeclaration, 'functions'],
  [SyntaxKind.InterfaceDeclaration, 'interfaces'],
  [SyntaxKind.EnumDeclaration, 'enums'],
  [SyntaxKind.VariableDeclaration, 'variables'],
  [SyntaxKind.VariableStatement, 'variables'],
  [SyntaxKind.PropertyDeclaration, 'properties'],
  [SyntaxKind.TypeAliasDeclaration, 'types'],
]);
```